MAUI/Kanban-Board/Column.md

```diff
@@ -90,33 +90,33 @@ Header shows the category [`Title`](), items count, min and max informations of
 
 {% highlight xaml %}
 
-     <kanban:SfKanban.HeaderTemplate >
-         <DataTemplate>
-             <StackLayout WidthRequest="300" HeightRequest="40"  BackgroundColor="Silver">
-                 <Label Margin="10" Text="{Binding Path=Title}" TextColor="Purple" HorizontalOptions="Start" />
-             </StackLayout>
-         </DataTemplate>
-     </kanban:SfKanban.HeaderTemplate>
+<kanban:SfKanban.HeaderTemplate >
+    <DataTemplate>
+        <StackLayout WidthRequest="300" HeightRequest="40"  BackgroundColor="Silver">
+            <Label Margin="10" Text="{Binding Path=Title}" TextColor="Purple" HorizontalOptions="Start" />
+        </StackLayout>
+    </DataTemplate>
+</kanban:SfKanban.HeaderTemplate>
 
 {% endhighlight %}
 
 {% highlight C# %} 
 
-    var headerTemplate = new DataTemplate(() => {
-        StackLayout root = new StackLayout() { 
-            WidthRequest = 300, 
-            HeightRequest = 40, 
-            BackgroundColor = Color.Silver 
-        };
-        Label label = new Label();
-        label.Margin = new Thickness(10); 
-        label.SetBinding(Label.TextProperty, new Binding("Title") );
-        label.TextColor = Color.Purple;
-        label.HorizontalOptions = LayoutOptions.Start; 
-        root.Children.Add(label);
-        return root;
-        });
-    kanban.HeaderTemplate = headerTemplate;
+var headerTemplate = new DataTemplate(() => {
+    StackLayout root = new StackLayout() { 
```

Comment:
@EzhilarasanElangovan31 code is not started at extreme left, may face alignment issue again

Reply:
aligned the code